src/auxiliary.jl

```diff
@@ -18,7 +18,8 @@ function resize_coeffs1!{T<:Number}(coeffs::Array{T,1}, order::Int)
     lencoef = length(coeffs)
     order ≤ lencoef-1 && return nothing
     resize!(coeffs, order+1)
-    coeffs[lencoef+1:end] .= zero(coeffs[1])
+    z = zero(coeffs[1])
+    @__dot__ coeffs[lencoef+1:order+1] = z
```

Comment:
It does the same as the simpler version. Or even do `coeffs[lencoef+1:end] .= 0`?